Samples/WorkflowCombineSampleApp/WorkflowCombineSampleAppUnitTests/WorkflowCombineSampleAppUnitTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  WorkflowCombineSampleAppUnitTests.swift
+//  WorkflowCombineSampleAppUnitTests
+//
+//  Created by Soo Rin Park on 11/1/21.
+//
+
+import Combine
+import Workflow
+import XCTest
+@testable import Development_WorkflowCombineSampleApp
+
+class DemoWorkerTests: XCTestCase {
+    func test_workflowIsRenderedEverySecondForFiveSeconds() {
+        let host = WorkflowHost(workflow: DemoWorkflow())
+
+        let expectation = XCTestExpectation(description: "host rendering is updated every second")
+        expectation.expectedFulfillmentCount = 5
+        let disposable = host.rendering.signal.observeValues { rendering in
+            print(rendering)
+            expectation.fulfill()
+        }
+
+        // buffer milisecond is added to account for the workflow to start running
+        wait(for: [expectation], timeout: 5.1)
```

Comment:
Can we re-write this test without the wait?

Reply:
Could you elaborate on why? The point of this test is to test that the publisher is firing every second per the Timer publisher which forces it to be asynchronous.

Reply:
Unit tests should be fast and abstracted away from external factors (like time). We can either inject a test scheduler we can control or use other mechanisms to not depend on the real delay.

Reply:
I can turn it into a snapshot test if you think that makes more sense

Reply:
I ended up just rewriting it to match the other sample app workflow tests